python/ray/rllib/policy/tf_policy_template.py

```diff
@@ -41,7 +41,7 @@ def build_tf_policy(name,
     This means that you can e.g., depend on any policy attributes created in
     the running of `loss_fn` in later functions such as `stats_fn`.
 
-    In eager mode (experimental), the following functions will be run
+    In eager mode (to be implemented), the following functions will be run
```

Comment:
Should we just remove the eager part from the doc string to not confuse anyone?

Reply:
I think this is probably fine, we are definitely going to add eager.